build.gradle

```diff
@@ -255,6 +255,7 @@ configure(javaProjects) { subproject ->
 		configurations.all {
 			resolutionStrategy.cacheChangingModulesFor 60, 'minutes'
 			exclude group: 'ch.qos.logback', module: 'logback-classic'
+			exclude group: 'org.projectreactor', module: 'reactor-core'
```

Comment:
@markpollack you can still then have a 1.1.x depedency only on core to satisfy the Environment object and exclude everything else (disruptor, sl4j, gs collection).